Singularity

```diff
@@ -0,0 +1,21 @@
+Bootstrap: yum
+OSVersion: 7
+MirrorURL: http://mirror.centos.org/centos-%{OSVERSION}/%{OSVERSION}/os/$basearch/
+Include: yum
```

Comment:
These 4 lines could be replaced with this:

```
BootStrap: docker
From: centos:7
```

You get a much more minimal environment but may speed up builds.

Reply:
I'll give that a try. Since we are loading almost everything from the host, we really don't need much environment in the guest.

Reply:
Ignoring the time it took to download the base boxes there doesn't seem to be much of a win here in terms of either disk usage in the final image or build time. I'm going to leave it as is.